src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_COUNT = 5;
+    private static final double DEFAULT_FIGURE_RADIUS = 10.0;
+
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int figureNumber = this.random.nextInt(this.FIGURE_COUNT);
```

Comment:
The 'this' keyword is not necessary here since 'FIGURE_COUNT' is a static constant. You can simply access it with 'FIGURE_COUNT' without 'this'.